Spiegel Online.js

```diff
@@ -39,64 +39,43 @@ http://www.spiegel.de/international/europe/0,1518,700530,00.html
 */
 
 function detectWeb(doc, url) {
-
-	var spiegel_article_XPath = './/div[@class="column-both"]/h2[@class="article-title"]|.//div[@class="column-wide"]/h2[contains(@class, "headline")]';
-	//the print edition is a magazine. Since the online edition is updated constantly it
-	//makes sense to treat it like a newspaper.
-	if (url.match(/\/print\//) && ZU.xpathText(doc, spiegel_article_XPath)){
-		return "magazineArticle";
-	}
-	else if (doc.evaluate(spiegel_article_XPath, doc, null, XPathResult.ANY_TYPE, null).iterateNext() ){ 
-		//Zotero.debug("newspaperArticle");
-		return "newspaperArticle";
-	} else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/thema/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	}  else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/suche/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	}  else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/international\/search/)){ 
-		//Zotero.debug("multiple");
-		return "multiple";
-	} else if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/international\/topic/)){ 
-		//Zotero.debug("multiple");
+	var spiegel_article_XPath = '//h2[@class="article-title"]';
+	if (doc.location.href.match(/^https?\:\/\/www\.spiegel\.de\/(thema|suche|international\/search|international\/topic)\// )){ 
```

Comment:
@aurimasv says that ```.search() != -1 ``` is more efficient than match, especially for complex expressions.